scss/index.scss

```diff
@@ -0,0 +1,107 @@
+@import "imports/variables";
+
+@font-face {
+  font-family: Ketchum;
+  src: url('../fonts/Ketchum.otf');
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: 'Cantarell', sans-serif;
+}
+
+main {
+  width: 1000px;
+  margin: 30px auto;
+}
+
+#scroll {
+  width: 100%;
+  height: 80vh;
+  margin: 0 auto;
+  overflow: auto;
+  border: 1px solid $poke-blue;
+  border-radius: 3px;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+h2 {
+  margin-bottom: 0;
+  margin-top: 2%;
+}
+
+.poke-container {
+  width: 100%;
+  height: 150px;
+  margin-bottom: 10px;
+}
+
+.info {
+  width: 65%;
+  height: 100%;
+  float: left;
+  background-color: $poke-grey;
+  padding-left: 5%;
+}
+
+.upper {
+  height: 30%;
+}
+.poke-title {
+  width: 70%;
+  float: left;
+}
+
+.poke-id {
+  width: 30%;
+  float: left;
+}
+
+.lower {
+  height: 70%;
+}
+
+.type-holder {
+  width: 20%;
+  float: left;
+  height: 100%;
+  font-size: 1.2rem;
+}
+
+.picture-holder {
+  float: left;
+  width: 35%;
+  height: 100%;
+  background-color: $poke-grey;
+}
+
+.picture {
+  height: 100%;
+  margin: auto;
+  display: block;
+}
+
+#save {
```

Comment:
Save button should not be included until a later story.

Reply:
Solved in d318df2

Reply:
Solved in d318df2